python/rascaline/rascaline/utils/power_spectrum.py

```diff
@@ -2,14 +2,34 @@
 from math import sqrt
 from typing import List, Optional, Union
 
-import numpy as np
-from equistore.core import Labels, TensorBlock, TensorMap
+from ._helpers import _dispatch
 
-from ..calculators import CalculatorBase
-from ..systems import IntoSystem
 
+# see rascaline-torch/utils/power_spectrum.py for an explanation of what's going on here
+try:
+    _ = CalculatorBase
+except NameError:
+    from ..calculator_base import CalculatorBase
 
-class PowerSpectrum:
+try:
+    _ = Labels
+    _ = TensorBlock
+    _ = TensorMap
+except NameError:
+    from equistore.core import Labels, TensorBlock, TensorMap
+
+try:
+    _ = IntoSystem
+except NameError:
+    from ..systems import IntoSystem
+
+try:
+    _ = torch_nn_module
+except NameError:
+    torch_nn_module = object
```

Comment:
All this should be removed and replaced by `from ._classes import Labels, TensorBlock, TensorMap, IntoSystem, CalculatorBase, torch_nn_module`. And then in `_classes.py` these are the pure Python versions of the classes; and in the manually created `rascaline.torch.utils._classes` module these are the TorchScript version of the classes.